spec/rails_helper.rb

```diff
@@ -48,3 +48,16 @@
 
   config.filter_rails_from_backtrace!
 end
+
+
+# Returns a hash of _specified_ (as opposed to runtime-adjusted/overridden) CSS attributes for an element
+def specified_style_attributes(capybara_element)
+  style_string = capybara_element['style']
+  attribute_strings = style_string.split(';')
+  attribute_strings.each_with_object({}) do |string, style_hash|
+    first_colon_position = string.index(':')
+    key = string[0...first_colon_position].strip
+    value = string[(first_colon_position+1)..-1].strip
+    style_hash[key] = value
+  end
+end
```

Comment:
@keithrbennett I would but this in a helper file in order to keep this file clean. You can create a new file in `spec/support`, and then just include it in the `rails_helper`.  For example:

```
# spec/support/css_helpers.rb
module CssHelpers
  def specified_style_attributes(capybara_element)
     ...
  end
end
```

Then add:

```
config.include CssHelpers
```

to the `rails_helper.rb` file.

Reply:
Perfect. This is exactly the advice I was seeking. I've made the change.